qmcpy/stopping_criterion/_cub_qmc_ld_g.py

```diff
@@ -156,3 +159,36 @@ def set_tolerance(self, abs_tol=None, rel_tol=None):
         """
         if abs_tol != None: self.abs_tol = abs_tol
         if rel_tol != None: self.rel_tol = rel_tol
+
+    def density_estimation(self):
```

Comment:
Can you clarify this a bit more? Me and Richard are confused as to what you exactly mean.

Reply:
Instead of returning the `kde` object for the continuous part, let the user pass in samples at which to evaluate the full KDE. Be careful as there is both the continuous part and discrete parts of the KDE.

Reply:
Updated. Pls check